staging/src/k8s.io/api/core/v1/annotation_key_constants.go

```diff
@@ -45,8 +45,10 @@ const (
 	// to one container of a pod.
 	SeccompContainerAnnotationKeyPrefix string = "container.seccomp.security.alpha.kubernetes.io/"
 
-	// SeccompProfileRuntimeDefault represents the default seccomp profile used by container runtime.
-	SeccompProfileRuntimeDefault string = "runtime/default"
+	// DeprecatedSeccompProfileRuntimeDefault represents the default seccomp profile used by container runtime.
+	// Seccomp annotations are now deprecated.
+	// This should be replaced by setting SeccompProfileRuntimeDefault on SeccompProfile.SeccompProfileType.
+	DeprecatedSeccompProfileRuntimeDefault string = "runtime/default"
```

Comment:
I assume it's okay to just rename the field as part of the deprecation policy, right?

Reply:
I am also assuming that, based on what was done when the docker default was deprecated. (commit: https://github.com/kubernetes/kubernetes/commit/e102633ae82b2a90e946c8bc6fbd99eb5dd0efb4)

Reply:
Yes, thank you for the clarification! :)